src/pages/Explorer/Layouts/SheetsTemplate.svelte

```diff
@@ -0,0 +1,49 @@
+<script>
+    import Svg from 'webkit/ui/Svg/svelte'
+
+    export let item;
+
+    let isOpen = false;
+</script>
+
+<div class="{isOpen ? 'open' : 'close'}" on:click={() => isOpen = !isOpen}>
+    <div class="row justify v-center">
+        <h5>{item.title}</h5>
+        <Svg id='arrow-down' w="10" class="$style.arrow" />
```

Comment:
1. According to figma design arrow has a 1width=101 and1 height=5.5`;
2. Apply `class="{isOpen ? '$style.open' : ''}"` directly to the `Svg` element.